Cargo.toml

```diff
@@ -11,7 +11,7 @@ readme = "README.md"
 vendored = ["openssl/vendored"]
 
 [target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies]
-security-framework = "2.0.0"
+security-framework = {version = "2.0.0", features = ["alpn"]}
```

Comment:
This will introduce a lower bound of macOS 10.13 - I wonder if it should be flagged as a Cargo feature in case people want to support older versions?

Reply:
Added a feature switch to make sure older MacOS versions are supported. I did enable the feature in the CI since that has an high enough MacOS version.